api/task.go

```diff
@@ -261,7 +260,7 @@ type TaskPatch struct {
 
 // TaskStatusPatch is the API message for patching a task status.
 type TaskStatusPatch struct {
-	ID int
+	IDList []int
```

Comment:
How about keep `TaskStatusPatch` as is, and change `PatchTaskStatus` to take `[]TaskStatusPatch`? This approach seems a more natural API.

Reply:
SGTM.